dulwich/archive.py

```diff
@@ -1,38 +1,43 @@
-# archive.py -- Creating an archive from a tarball
-# Copyright (C) 2015 Jonas Haag <jonas@lophus.org>
-# Copyright (C) 2015 Jelmer Vernooij <jelmer@jelmer.uk>
-#
-# Dulwich is dual-licensed under the Apache License, Version 2.0 and the GNU
-# General Public License as public by the Free Software Foundation; version 2.0
-# or (at your option) any later version. You can redistribute it and/or
-# modify it under the terms of either of these two licenses.
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-# You should have received a copy of the licenses; if not, see
-# <http://www.gnu.org/licenses/> for a copy of the GNU General Public License
-# and <http://www.apache.org/licenses/LICENSE-2.0> for a copy of the Apache
-# License, Version 2.0.
-#
-
 """Generates tarballs for Git trees.
 
+archive.py -- Creating an archive from a tarball
+Copyright (C) 2015 Jonas Haag <jonas@lophus.org>
+Copyright (C) 2015 Jelmer Vernooij <jelmer@jelmer.uk>
+
+Dulwich is dual-licensed under the Apache License, Version 2.0 and the GNU
+General Public License as public by the Free Software Foundation; version 2.0
+or (at your option) any later version. You can redistribute it and/or
+modify it under the terms of either of these two licenses.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+You should have received a copy of the licenses; if not, see
+<http://www.gnu.org/licenses/> for a copy of the GNU General Public License
+and <http://www.apache.org/licenses/LICENSE-2.0> for a copy of the Apache
+License, Version 2.0.
+
 """
 
 import posixpath
+
```

Comment:
I added a blank line because of the Python specification.

Reply:
I don't think it calls for empty lines in between imports?

Reply:
If i have this style:

```Python
import sys
from os.path import basename

from getopt import getopt

from dulwich import porcelain
```

A message like this will appear:

```
H306: imports not in alphabetical order (sys, os.path.basename)
```

This is in accordance with Flake8's specification.

Reply:
That's just about the ordering, not about adding extra lines.

Reply:
OK